common/src/services/stateMigration.service.ts

```diff
@@ -132,17 +132,12 @@ export class StateMigrationService {
   ) {}
 
   async needsMigration(): Promise<boolean> {
-    const currentStateVersion = (
-      await this.storageService.get<GlobalState>(keys.global, {
-        htmlStorageLocation: HtmlStorageLocation.Local,
-      })
-    )?.stateVersion;
+    const currentStateVersion = await this.getCurrentStateVersion();
```

Comment:
This call no longer passes the StorageOptions to retrieve GlobalState. Can this be omitted?

Reply:
This ends up back in a this.get() call that handles passing in correct options.

Reply:
Ah I see, missed that 😄